### DISCUSSION TITLE
Pixelate 3D

### DISCUSSION THREAD

**Discussion by DasLixou** [bevyengine/bevy#6632] [Q&A] [answered]
Is there a easy way to pixelate a 3D Camera by just setting the pixels lower or ideally a simple shader script? thanks :)

**Comment by torsteingrindvik** [comment, marked as answer]
I have a plugin which supports this: https://github.com/torsteingrindvik/bevy-vfx-bag
If you scroll down there is a video of pixelation with an adjustable block size.

If you don't want to use the plugin you could take a look at the shader: https://github.com/torsteingrindvik/bevy-vfx-bag/blob/main/assets/shaders/pixelate.wgsl
and add it to your project.

**Reply by DasLixou** [reply]
i get this error: 
```
error[E0277]: the trait bound `BevyVfxBagPlugin: bevy::prelude::Plugin` is not satisfied
   --> src/camera.rs:22:24
    |
22  |         app.add_plugin(BevyVfxBagPlugin) // This needs to be added for any effect to work
    |             ---------- ^^^^^^^^^^^^^^^^ the trait `bevy::prelude::Plugin` is not implemented for `BevyVfxBagPlugin`
    |             |
    |             required by a bound introduced by this call
    |
    = help: the following other types implement trait `bevy::prelude::Plugin`:
              AnimationPlugin
              AssetCountDiagnosticsPlugin<T>
              AssetPlugin
              AudioPlugin
              BloomPlugin
              CameraProjectionPlugin<T>
              ColorMaterialPlugin
              Core2dPlugin
            and 46 others
```

**Reply by torsteingrindvik** [reply]
I think this usually happens if the Bevy dependency of the plugin and your Bevy dependency differs.
Are you using Bevy 0.9 or main?

EDIT: I saw you got your answer in https://github.com/bevyengine/bevy/discussions/6650

**Reply by DasLixou** [reply]
Yes Sorry 😅 but I got another problem and created an issue on your repo :)
